Frameworks/UIKit/UIImage.mm

```diff
@@ -889,22 +892,34 @@ + (UIImage*)animatedResizableImageNamed:(NSString*)name capInsets:(UIEdgeInsets)
     return StubReturn();
 }
 
+static NSData* _ObtainImageRepresentation(UIImage* img, CFStringRef type, NSDictionary* properties) {
+    RETURN_NULL_IF(!img);
+    CFMutableDataRef cfData = CFDataCreateMutable(nullptr, 0);
+    CFAutorelease(cfData);
+    CGImageDestinationRef destination = CGImageDestinationCreateWithData(cfData, type, 1, nullptr);
+    CFAutorelease(destination);
+    CGImageDestinationAddImage(destination, getImage(img), static_cast<CFDictionaryRef>(properties));
+    if (!CGImageDestinationFinalize(destination)) {
+        return nil;
+    }
+    return static_cast<NSData*>(cfData);
+}
+
 /**
- @Status Stub
+ @Status Interoperable
 */
 NSData* UIImagePNGRepresentation(UIImage* img) {
```

Comment:
yep adding, the ping was for the rest of the reviewers. 

---
In reply to: [96518605](https://github.com/Microsoft/WinObjC/pull/1704#discussion_r96518605) [](ancestors = 96518605,96090810)